contracts/clients/src/Aggregator.ts

```diff
@@ -38,12 +39,18 @@ export type BundleReceiptError = {
   submitError: string | undefined;
 };
 
-export type BundleReceipt = {
-  transactionIndex: number;
-  transactionHash: string;
+/**
+ * The BLS Wallet specific values in a {@link BundleReceipt}.
+ */
+export type BlsWallet = {
```

Comment:
Can't decide if I like this name, any suggestions welcome. 

Few random ideas:
`BlsWalletReceipt`
`BlsTransactionReceipt`
`BlsBundleReceipt`
`BundleTransactionsReceipt`
`Bundles`
`BlsBundles`

Reply:
Yeah I agree, I think it would be good to have the word receipt in the name.  Maybe something like BlsBundleReceipt sounds good 👍

Reply:
BlsBundleReceipt sounds good to me, will update